pilot/cmd/pilot-agent/main.go

```diff
@@ -268,9 +268,9 @@ func timeDuration(dur *duration.Duration) time.Duration {
 func init() {
 	proxyCmd.PersistentFlags().StringVar((*string)(&registry), "serviceregistry",
 		string(serviceregistry.KubernetesRegistry),
-		fmt.Sprintf("Select the platform for service registry, options are {%s, %s, %s, %s, %s}",
+		fmt.Sprintf("Select the platform for service registry, options are {%s, %s, %s, %s, %s, %s}",
```

Comment:
Very interesting - can you check if this is used in any place in the agent code ? File a bug for 1.0 - the agent should have no dependency and use no registry-related code

Reply:
The value is used poorly here: https://github.com/istio/istio/blob/8fd9e21e84eeb941a05967f1bbb791ec46a0c685/pilot/cmd/pilot-agent/main.go#L92

So there's not a real dependency on the service registry ... and adding the new option here is really just for consistency.